### ISSUE TITLE
[Feature-Request] Fetch Tweet and check, verification time

### ISSUE THREAD

**Issue by Ankuraxz** [umanghome/twitter-search-covid19#23]
The Covid resources come and get shared/ utilized within a matter of minutes, hence contacts verified around 2 hours ago are obsolete, specifically for large cities like Delhi. I request a feature to fetch and check the verification time.

**Comment by harshil15999** [comment]
@Ankuraxz please elaborate slightly

**Comment by Ankuraxz** [comment]
Some Volunteers put resources, which have been verified 3-4 hours ago. Some volunteers RT them or copy and re-paste these. What happens is, whenever we search a particular resource through Twitter (or your platform), these types of obsolete tweets come at the top. Since the demand is huge, these tweets are generally obsolete and hence, the mentioned contact is either switched off or busy as the resource is exhausted.

**Comment by harshil15999** [comment]
@Ankuraxz basically you want a timestamp filtering?

**Comment by Ankuraxz** [comment]
Not exactly, timestamps of tweet represent when the tweet was put up. I am talking about the time when a particular resource was verified. Here is an example. 
![Untitled design (6)](https://user-images.githubusercontent.com/34418414/116032016-899f3500-a67c-11eb-91be-dc995d996bab.png)

Its also important to understand the type and quantity of resource. 1 ICU can get used in the matter of minutes, whereas 1500 beds may take days to get acquired.
